lib/Doctrine/DBAL/Exception/MissingSQLParam.php

```diff
@@ -12,7 +12,7 @@ final class MissingSQLParam extends SQLParserUtilsException
     public static function new(string $paramName) : self
     {
         return new self(
-            sprintf('Value for :%1$s not found in params array. Params array key should be "%1$s"', $paramName)
+            sprintf('Value for :%1$s not found in params array. Params array key should be "%1$s"', $paramName) // what is this syntax?
```

Comment:
It allows reusing one parameter multiple times by its position. See [`sprintf()`](https://php.net/sprintf#example-6099).